ecosystem/typescript/sdk_v2/src/crypto/authentication_key.ts

```diff
@@ -50,21 +51,12 @@ export class AuthenticationKey {
     return this.data.toUint8Array();
   }
 
-  /**
-   * Converts a K-of-N MultiEd25519PublicKey to AuthenticationKey with:
-   * `auth_key = sha3-256(p_1 | … | p_n | K | 0x01)`. `K` represents the K-of-N required for
-   * authenticating the transaction. `0x01` is the 1-byte scheme for multisig.
-   *
-   * @param multiPublicKey A K-of-N MultiPublicKey
-   * @returns AuthenticationKey
-   */
-  static fromMultiPublicKey(args: { multiPublicKey: MultiEd25519PublicKey }): AuthenticationKey {
-    const { multiPublicKey } = args;
-    const multiPubKeyBytes = multiPublicKey.toUint8Array();
-
-    const bytes = new Uint8Array(multiPubKeyBytes.length + 1);
-    bytes.set(multiPubKeyBytes);
-    bytes.set([AuthenticationKey.MULTI_ED25519_SCHEME], multiPubKeyBytes.length);
+  static fromBytesAndScheme(args: { hexInput: HexInput; scheme: AuthenticationKeyScheme }) {
```

Comment:
Should we name this from `hexInput` to `bytes` in order to match the name `fromBytesAndScheme`?